elide-core/src/main/java/com/yahoo/elide/resources/JsonApiEndpoint.java

```diff
@@ -112,13 +113,29 @@ public Response patch(
      */
     @DELETE
     @Path("{path:.*}")
+    @Consumes("application/vnd.api+json")
     public Response delete(
         @PathParam("path") String path,
         @Context SecurityContext securityContext,
         String jsonApiDocument) {
         return build(elide.delete(path, jsonApiDocument, getUser.apply(securityContext)));
     }
 
+    /**
+     * Delete resource handler (expects no body and no content-type header).
+     *
+     * @param path request path
+     * @param securityContext security context
+     * @return response
+     */
+    @DELETE
+    @Path("{path:.*}")
+    public Response deleteWithoutBody(
```

Comment:
@clayreimann I don't see any reason to entirely freeze any and all patches in waiting for elide 3.0 (at least until we work on releasing it full bore). Unless anyone else has different thoughts, I suspect we'll continue to patch elide 2.x for the near future :)

Anyway, to avoid confusion, can we try something like the following (untested, just an idea of some variation like this):

```
    @DELETE
    @Path("{path:.*}")
    @Consumes({"", "application/vnd.api+json"})
    public Response delete(
        @PathParam("path") String path,
        @Context SecurityContext securityContext,
        @DefaultValue("") String jsonApiDocument) {
        String doc = (jsonApiDocument.isEmpty()) ? null : jsonApiDocument;
        return build(elide.delete(path, doc, getUser.apply(securityContext)));
    }
```